level.go

```diff
@@ -0,0 +1,51 @@
+package lsm
+
+const (
+	maxLevel   = 7
+	l0Capacity = 5 // 5개 생기면 l0 compaction
+	growFactor = 10
+)
+
+type level struct {
+	sstables []*SSTable // SSTables in this level
+}
+
+func (l *level) sstableToCompact() *SSTable {
+	// 현재 레벨에서 가장 오래된 파일 반환
+	if len(l.sstables) == 0 {
+		return nil
+	}
+	minSSTable := l.sstables[0]
+	for _, sstable := range l.sstables {
+		if sstable.file.Name() < minSSTable.file.Name() {
+			minSSTable = sstable
+		}
+	}
+	return minSSTable
+}
+
+func (l *level) TotalSize() int {
+	totalSize := 0
+	for _, sstable := range l.sstables {
+		f, _ := sstable.file.Stat()
```

Comment:
### Runtime Errors

- **Potential Nil Pointer Dereference**: In `TotalSize`, ensure that `sstable.file` is not nil before calling `Stat` on it. This can be done by adding a nil check:

```go
if sstable.file == nil {
    continue
}
f, err := sstable.file.Stat()
if err != nil {
    return 0
}
totalSize += int(f.Size())
```